lib/transform/transformers.js

```diff
@@ -30,11 +30,10 @@ export function assets (asset) {
   transformedAsset.fields.file = reduce(
     asset.fields.file,
      (newFile, file, locale) => {
-       newFile[locale] = omit(file, 'url', 'details')
-       if (newFile[locale].uploadFrom) {
-         delete newFile[locale].uploadFrom
+       if (!newFile[locale].uploadFrom) {
```

Comment:
This won't work  because`newFile` starts as is an empty object [1] for each locale. So `newFile[locale]` will always be `undefined`.

This should have been caught by a test. If there's no test we should write it.

[1] https://github.com/contentful/contentful-batch-libs/pull/94/files#diff-38a92f6d9573a0032767c762b9e9e235L40